app/src/main/kotlin/fr/o80/mostwanted/component/solution/S03Screen.kt

```diff
@@ -0,0 +1,50 @@
+package fr.o80.mostwanted.component.solution
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun S03Screen() {
+    Scaffold(
+        topBar = {
+            CenterAlignedTopAppBar(
+                title = {
+                    Text(text = "Best People")
+                }
+            )
+        }
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(4.dp)
+            ) {
+                Text(
+                    text = "Prénom"
+                )
+                Text(
+                    text = "Nom"
+                )
+            }
```

Comment:
Ca te dit qu'on se fasse mousser ici ?

```suggestion
            Row(
                horizontalArrangement = Arrangement.spacedBy(4.dp),
                verticalAlignment = Alignment.CenterVertically
            ) {
                Text(text = "Audrey")
                Icon(Icons.Default.Star, contentDescription = null)
                Text(text = "Olivier")
            }
```

![image](https://github.com/olivierperez/MostWanted/assets/883959/abdefb05-e5c9-43a8-becb-31fd413eca06)